templates/Web/Pages/React.List/src/components/ReactList/ListForm.jsx

```diff
@@ -12,9 +12,9 @@ export default function ListForm(props) {
         placeholder="Add text here..."
         aria-label="Add text here..."
       />
-      <button className="btn btn-primary ml-2" onClick={onAddListItem}>
+      <button className="btn btn-primary ml-2">
```

Comment:
I think it's better to explicitly define type="submit" on the button. It's default for most of the browsers, but per W3Schools: "Always specify the type attribute for the <button> element. Different browsers may use different default types for the <button> element."
https://www.w3schools.com/tags/att_button_type.asp

Reply:
That's a good catch. I will update that. Thanks!

Reply:
Really good catch!